app/code/community/Bolt/Boltpay/Model/Setup.php

```diff
@@ -0,0 +1,37 @@
+<?php
+/**
+ * Bolt magento plugin
+ *
+ * NOTICE OF LICENSE
+ *
+ * This source file is subject to the Open Software License (OSL 3.0)
+ * that is bundled with this package in the file LICENSE.txt.
+ * It is also available through the world-wide-web at this URL:
+ * http://opensource.org/licenses/osl-3.0.php
+ *
+ * @category   Bolt
+ * @package    Bolt_Boltpay
+ * @copyright  Copyright (c) 2019 Bolt Financial, Inc (https://www.bolt.com)
+ * @license    http://opensource.org/licenses/osl-3.0.php  Open Software License (OSL 3.0)
+ */
+
+class Bolt_Boltpay_Model_Setup extends Mage_Eav_Model_Entity_Setup
+{
+    /**
+     * Run module modification files. Return version of last applied upgrade (false if no upgrades applied)
+     *
+     * @param string $actionType self::TYPE_*
+     * @param string $fromVersion
+     * @param string $toVersion
+     * @return string|false
+     * @throws Mage_Core_Exception
+     */
+    protected function _modifyResourceDb($actionType, $fromVersion, $toVersion)
```

Comment:
@mccottry What do you think about test for this method?
We couldn't mock parent call, so we can:
- don't create for this
- call _modifyResourceDb with parameters (<type>,'0.0.1','0.0.2') so parent call will not do any work because it doesn't  find matching files

Reply:
The stubbed parameters of the second approach seems more desirable.

Reply:
Done